packages/ios-client/Sources/FishjamClient/webrtc/PeerConnectionManager.swift

```diff
@@ -176,11 +176,9 @@ internal class PeerConnectionManager: NSObject, RTCPeerConnectionDelegate {
         config.candidateNetworkPolicy = .all
         config.tcpCandidatePolicy = .disabled
 
-        // if ice servers are not empty that probably means we are using turn servers
+        // if ice servers are not empty that probably means we are using turn servers, which are not handled at the moment
```

Comment:
**suggestion:** 

We might actually start using turn servers, that will be passed in the `connected` message. They will be handled out of the box
```suggestion
```